Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN make build
 FROM gcr.io/distroless/static:debug
 
 WORKDIR /
-COPY --from=builder /workspace/bin/k8s .
+COPY --from=builder /workspace/bin/plainv0 .
```

Comment:
I know this is probably annoying, but wdyt about `plain-v0` instead, just to make the name/version distinction more obvious?

Reply:
That sounds reasonable. We can still use the name as an import path and everywhere else, so I think it's a good change. Will update.